pkgs/by-name/vi/virter/package.nix

```diff
@@ -25,6 +30,30 @@ buildGoModule rec {
     "-X github.com/LINBIT/virter/cmd.githash=builtByNix"
   ];
 
+  nativeBuildInputs = [
+    makeWrapper
+    installShellFiles
+  ];
+
+  postInstall = ''
+    wrapProgram $out/bin/virter \
+      --prefix PATH ":" ${lib.makeBinPath [ dnsmasq ]}
+  ''
+  + lib.optionalString (stdenv.hostPlatform.emulatorAvailable buildPackages) (
```

Comment:
Use of an emulator for this is (possibly) discouraged, see https://github.com/NixOS/nixpkgs/pull/422712 ; consider just guarding the execution instead https://nixos.org/manual/nixpkgs/stable/#installshellfiles-installshellcompletion-exampleusage-guarded .